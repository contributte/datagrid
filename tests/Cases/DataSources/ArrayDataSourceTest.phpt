<?php

namespace Ublaboo\DataGrid\Tests\Cases\DataSources;
use Ublaboo;

require __DIR__ . '/BaseDataSourceTest.phpt';


final class ArrayDataSourceTest extends BaseDataSourceTest
{

	public function setUp()
	{
		$this->ds = new Ublaboo\DataGrid\DataSource\ArrayDataSource($this->data);
		$factory = new Ublaboo\DataGrid\Tests\Files\XTestingDataGridFactory;
		$this->grid = $factory->createXTestingDataGrid();
	}
}


$test_case = new ArrayDataSourceTest;
$test_case->run();
