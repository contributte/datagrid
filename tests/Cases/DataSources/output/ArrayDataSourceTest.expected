[
	[
		'id' => 3,
		'name' => 'Santa Claus',
		'age' => 12,
		'address' => 'New York',
	],
]
